src/backend/src/utils/work-packages.utils.ts

```diff
@@ -25,6 +25,9 @@ export const getWbsElementId = async ({
       wbsNumber: { carNumber, projectNumber, workPackageNumber }
     }
   });
+
+  if (wbsElem?.dateDeleted) return undefined;
```

Comment:
I don't know much about this function. Does returning undefined match the expected behavior by consumer code? Does returning undefined also match the behavior of prisma.findUnique in the case where the WBS element is not found? just wanna be sure.

Reply:
i dont like this function at all but this is how it was before so i just went with it

Reply:
actually it's only used in one spot so im gonna change it cuz it sucks